tests/schemas/test_openapi.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 from django.conf.urls import url
 from django.test import RequestFactory, TestCase, override_settings
+from django.utils.translation import ugettext_lazy as _
```

Comment:
This can just be `gettext_lazy` now, since we dropped Python 2, and the `u` versions are thereby unnecessary (and ≈deprecated I guess).

Reply:
alright, fixed now